### PR TITLE
Add docker install

### PR DESCRIPTION
**Pull request by tomberek**

### CODE REVIEW

**Comment by tomberek** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by tomberek** [comment]
recheck

File: source/tutorials/install-nix.rst [hunk 1]
```diff
@@ -25,3 +25,20 @@ Check that the installation was successful:
...
+.. code:: bash
+
+    $ docker run -it nixos/nix
+
+Or start a Docker shell with Nix exposing a work directory:
+
+.. code:: bash
+
+    $ mkdir workdir
+    $ docker run -it -v $(pwd)/workdir:/root nixos/nix
```
Comment (domenkozar) [inline source/tutorials/install-nix.rst:43]: Any particular reason not to mount it at /workdir? To me /root feels abusing the convention of what should be there

Comment (tomberek) [reply source/tutorials/install-nix.rst]: no reason, moved over to `/workdir`

File: source/tutorials/install-nix.rst [hunk 2]
```diff
@@ -25,3 +25,20 @@ Check that the installation was successful:
...
+Use Docker to explore Nix
+-------------------------
+
+Start a Docker shell with Nix:
+
+.. code:: bash
+
+    $ docker run -it nixos/nix
+
+Or start a Docker shell with Nix exposing a work directory:
```
Comment (domenkozar) [inline source/tutorials/install-nix.rst:38]:
```suggestion
Or start a Docker shell with Nix exposing a ``workdir`` directory:
```

**Comment by domenkozar** [comment]
Thanks!

**Pull request merged**